docs/manuals/source/IntroductionAndTutorial/WhatIsBareos.rst

```diff
@@ -164,11 +164,11 @@ Not all packages are required to run Bareos.
 
 .. _section-BareosBinaryReleasePolicy:
 
-.. _section-BareosRepositoryTypes:
-
 Bareos Binary Release Policy
 ----------------------------
 
+.. _section-BareosRepositoryTypes:
```

Comment:
Why have moved this below the section title? IMHO it belongs above the title. Multiple section labels are not a problem.

Reply:
Don't know what was not working, restested and it works. Would you like to see this title going back before the section?

Reply:
Either move it above the section title (as this ref is also called "section-") or remove it completely, as it is not used anymore anyway.